docs/source/markdown/podman-systemd.unit.5.md

```diff
@@ -1145,6 +1146,17 @@ performance and robustness reasons.
 The format of the name is the same as when passed to `podman pull`. So, it supports using
 `:tag` or digests to guarantee the specific image version.
 
+### `ImageTag=`
+
+The name of the image after being pulled.
+This parameter should be used when the value of `Image` is not the name of the resulting image.
+
+For example, an image that was saved into a `docker-archive` with the following command:
+
+`podman image save --format docker-archive --output /tmp/archive-file.tar quay.io/podman/stable:latest`
+
+will require setting `Image=docker-archive:/tmp/archive-file.tar` and `ImageTag=quay.io/podman/stable:latest`
```

Comment:
IDK if this is closer, I'm struggling with the wording myself.
```suggestion
would require setting the value of the `Image` and `ImageTag` systemd configuration options to `Image=docker-archive:/tmp/archive-file.tar` and `ImageTag=quay.io/podman/stable:latest`
```

Reply:
requires setting the `Image=docker-archive:/tmp/archive-file.tar` and `ImageTag=quay.io/podman/stable:latest`

Take the `will` and `would` out of it, and adding the image and imagetag names twice does not lead to clarity.